src/content/docs/style-guide/writing-docs/processes-procedures/create-preview-document.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Create content for the preview docs site
+contentType: page
+template: basicDoc
+topics:
+  - Basic style guide
+  - Writing guidelines
+redirects:
+---
+
+The preview docs site hosts preview docs for upcoming features that we share with customers.
```

Comment:
I can think you can safely remove the second "preview" in this sentence :)

Reply:
I think one thing that would be nice to have in this first paragraph or section is what someone will get out of taking the time to read this whole doc. By the end, do we want people do be able to contribute to the preview doc? Do we want people to understand what the preview docs site is? What do we want people to know about it?

We can do our readers a kindness by giving them a sense right at the start about what they'll get out of spending their time (a non-renewable resource!) reading this.